integration/compiler-test.ts

```diff
@@ -165,7 +165,7 @@ test.describe("compiler", () => {
   test("removes server code with `*.server` files", async ({ page }) => {
     let app = new PlaywrightFixture(appFixture, page);
     let res = await app.goto("/", true);
-    expect(res.status()).toBe(200); // server rendered fine
+    expect(res?.status()).toBe(200); // server rendered fine
```

Comment:
Thanks for this! Is there any way we can make it so `app.goto` always returns a `res`? I can't think of a situation where it'd be ok if it didn't 😬

Reply:
We're just re-using `playwright`'s `app.goto` under the hood, which is returning `null | Response`.

```ts
  /**
   * Returns the main resource response. In case of multiple redirects, the navigation will resolve with the response of the
   * last redirect.
   *
   * [...]
   * 
   * > NOTE: The method either throws an error or returns a main resource response. The only exceptions are navigation to
   * `about:blank` or navigation to the same URL with a different hash, which would succeed and return `null`.
   *
   * [...]
   */
```

Since we're not going to use `about:blank`, I think we can make our helper function just always return `Response`

Reply:
Done